src/store/modules/user/actions.ts

```diff
@@ -18,6 +18,7 @@ const actions: ActionTree<UserState, RootState> = {
       const resp = await UserService.login(username, password)
       if (resp.status === 200 && resp.data) {
         if (resp.data.token) {
+          dispatch('setDateTimeFormat', process.env.VUE_APP_DATE_FORMAT);
```

Comment:
We can have a fallback format when env file does not contain a date time format.

Reply:
Added fallback format